Standards/scs-0111-v1-volume-type-decisions.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Decisions for the Volume Type Standard
+type: Decision Record
+status: Draft
+track: IaaS
+---
+
+## Introduction
+
+Volume Types are used to classify volumes and provide a basic decision for what kind of volume should be created. These volume types can sometimes very be backend-specific and it might be hard for a user to choose the most suitable volume type, if there is more than one default type.
```

Comment:
We should add a definition of what a "volume" is from our point of view.

Reply:
@josephineSei Reading over this and the comment from @berendt: In the one session we did on this (6th of July) we did discuss this iirc (since in that session @jnull raised this question).